ui/src/Common/FormNotesTextArea/FormNotesTextArea.test.tsx

```diff
@@ -22,13 +22,13 @@ import FormNotesTextArea from './FormNotesTextArea';
 describe('notes input', () => {
     it('should count the number of characters', () => {
         const callback = jest.fn();
-        const notes = render(<FormNotesTextArea notes={'this string is 17'} callBack={callback}/>);
+        const notes = render(<FormNotesTextArea note={'this string is 17'} onChange={callback}/>);
```

Comment:
👏 `onChange` is a much better name than `callBack` because it's no longer and is more precise